src/Components/Components/src/RenderTree/Renderer.cs

```diff
@@ -1079,7 +1091,7 @@ public void Dispose()
         /// <inheritdoc />
```

Comment:
I know it's not the topic of this PR, however since we are already updating this, I would like to take the chance to suggest that we mark Dispose here as obsolete and remove it in the future. The renderer is an "internal" API and  for all intents and purposes we are the only ones (with a few exceptions) using it.

The renderer implementing IDisposable is a remnant of olden times when IAsyncDisposable didn't exist, and our programming model is highly skewed towards async computing, so we should take the opportunity now to start obsoleting these and avoid having to worry about them in the future.

Reply:
I'd be fine with us doing this, but don't want to make it part of this PR as there's relatively little time for me to do the remaining things that need to be done this week. Do you want to file an issue? Maybe we can do it in RC2.